test/ejabberd_tests/tests/connect_SUITE.erl

```diff
@@ -57,6 +60,13 @@ groups() ->
                      invalid_stream_namespace]},
      {pre_xmpp_1_0, [], [pre_xmpp_1_0_stream]},
      {starttls, test_cases()},
+     {feature_order, [stream_features_test,
```

Comment:
As we are in process of adopting parallel tests I strongly recommend using the `parallel` property for this group and `escalus_fresh` module. You can take a look at this SUITE [1] for inspiration how to use it (also outside story - test case `server_announces_csi`).

[1]. https://github.com/esl/MongooseIM/blob/684bc0fbcab4c6e298961bcb7a0147c2b0678c0a/test/ejabberd_tests/tests/xep_0352_csi_SUITE.erl